### DISCUSSION TITLE
Univision West | 41 New York (WXTV-DT1)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#497] [Show and tell]
### Channel ID

WXTVDT1.us

### Stream URL (optional)

https://unidfp-nlds143.global.ssl.fastly.net/nlds/univisionnow/univision_ny2/as/live/univision_ny2_hd_pc.m3u8?t1=

### Notes (optional)

Don't know if this the new york version of Univision 
but I was changing the unidfp-nlds number and the univision_(cityname)2 for example:
138 = univison_dal2 Is for 23 KUVN Dallas
145 = univison_hou2 Is for 45 KXLN Houston 
154 = univision_chi2 Is for Univision Chicago
So 143 = univison_ny2 might be for Univision 41 New York
iptv-org/iptv#7789 iptv-org/iptv#13710
EDIT: This is the New york version of Univision 
^Sorry for low quality version screenshot
![Screenshot_20230803_204235_Chrome](https://github.com/iptv-org/iptv/assets/83226646/11a449d0-72f8-419f-95d3-5b7b0906e6cf)

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://unidfp-nlds143.global.ssl.fastly.net/nlds/univisionnow/univision_ny2/as/live/univision_ny2_hd_pc.m3u8?t1= (offline)
<img width="804" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/247f75ee-c7e7-4c30-b01f-f73f96814c05">

**Reply by R4IN2007** [reply]
Yeah, The West version of Univision (Dallas, New York) aren't working only the East version of Univision (Houston, Chicago) are working
